graph-builder/tests/net/quay_io/mod.rs

```diff
@@ -73,6 +76,16 @@ fn expected_releases(
     releases
 }
 
+fn remove_metadata_by_key(releases: &mut Vec<Release>, key: &str) -> Vec<Release> {
+    releases
+        .iter_mut()
+        .map(|release| {
+            release.metadata.metadata.remove(key).unwrap();
+            release.clone()
```

Comment:
If there is a way to avoid both the `unwrap` and the `clone` in this closure, that would be better.